### PR TITLE
chore: improve linter

### PR DESCRIPTION
**Pull request by septs**

### CODE REVIEW

File: packages/.eslintrc.json [hunk 1]
```diff
@@ -54,6 +54,8 @@
         "unicorn/prefer-number-properties": "error",
         "unicorn/throw-new-error": "error",
         "unused-imports/no-unused-imports-ts": "error",
+        "@dimensiondev/no-bigint": "error",
+        "@dimensiondev/no-invisible-characters": "error",
```
Comment (Jack-Works) [inline packages/.eslintrc.json]: Is this a solution to CVE-2021-42574 and CVE-2021-42694?

File: packages/mask/src/extension/background-script/Jobs/InjectContentScripts.ts [hunk 2]
```diff
@@ -53,7 +53,7 @@ export default function (signal: AbortSignal) {
         signal.addEventListener(
             'abort',
             MaskMessages.events.maskSDKHotModuleReload.on(async () => {
-                const code = (await fetchUserScript(maskSDK_URL)) + `\n;console.log("[@masknet/sdk] SDK reloaded.")`
+                const code = (await fetchUserScript(maskSDK_URL)) + '\\n;console.log("[@masknet/sdk] SDK reloaded.")'
```
Comment (Jack-Works) [inline packages/mask/src/extension/background-script/Jobs/InjectContentScripts.ts]: I think this change is wrong.

File: packages/mask/src/plugins/Trader/constants/uniswap.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @dimensiondev/no-bigint */
```
Comment (Jack-Works) [inline packages/mask/src/plugins/Trader/constants/uniswap.ts]: Seems like your rule has a false-positive. Only `BigInt(1)` or `1n` is the usage of bigint, `JSBI.BigInt(50)` is not

File: packages/plugins/Flow/src/constants.ts [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { ChainId, NetworkType, ProviderType } from '@masknet/web3-shared-flow/ty
 export const PLUGIN_ID = PluginId.Flow
 export const PLUGIN_META_KEY = `${PluginId.Flow}:1`
 export const PLUGIN_NAME = 'Flow Chain'
-export const PLUGIN_ICON = '⚙️'
+export const PLUGIN_ICON = '⚙'
```
Comment (Jack-Works) [inline packages/plugins/Flow/src/constants.ts]: Why those emoji changes? Any invisible Unicode (like Zero-Width-Joiner) involved?

Comment (septs) [reply packages/plugins/Flow/src/constants.ts]: https://unicode-table.com/en/FE0F/

**Review by Jack-Works — DISMISSED** [review]

**Review by guanbinrui — DISMISSED** [review]

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**